README.md

```diff
@@ -1,22 +1,63 @@
-# Ionic Framework
-
-[Ionic Framework](https://ionicframework.com/) is the open-source mobile app development framework that makes it easy to
-build top quality native and progressive web apps with web technologies.
-
-Ionic Framework is based on [Web Components](https://www.webcomponents.org/introduction) and comes with many significant performance, usability, and feature improvements over the past versions.
-
-## Looking for the Ionic Framework v6 beta?
-
-[Click here to get started!](https://github.com/ionic-team/ionic-framework/blob/next/BETA.md)
+<p align="center">
+  <a href="#">
+    <img alt="Ionic" src="https://ionic.io/_next/image?url=https%3A%2F%2Fimages.prismic.io%2Fionicframeworkcom%2F66cfdbef-e59d-463a-8e24-12cb233e9d97_ionic%2Blogo%2Bblue.png&w=128&q=75" width="60" />
+  </a>
+</p>
+
+<h1 align="center">
+  Ionic
+</h1>
+
+<p align="center">
+  Ionic is an open source app development toolkit for building modern, fast, top-quality cross-platform native and Progressive Web Apps from a single codebase with JavaScript and the Web.
+</p>
+<p align="center">
+  Ionic is based on <a href="https://www.webcomponents.org/introduction">Web Components</a>, which enables significant performance, usability, and feature improvements alongside support for popular web frameworks like <a href="https://angular.io/">Angular</a>, <a href="https://reactjs.com/">React</a>, and <a href="https://vuejs.org/">Vue</a>.
+
+</p>
+
+<p align="center">
+  <a href="https://github.com/ionic-team/ionic-framework/blob/main/LICENSE">
+    <img src="https://img.shields.io/badge/license-MIT-blue.svg" alt="Ionic Framework is released under the MIT license." />
+  </a>
+  <a href="https://ionicframework.com/docs/contributing/how-to-contribute">
+    <img src="https://img.shields.io/badge/PRs-welcome-brightgreen.svg" alt="PRs welcome!" />
+  </a>
+  <a href="https://twitter.com/Ionicframework">
+    <img src="https://img.shields.io/twitter/follow/ionicframework.svg?label=Follow%20@IonicFramework" alt="Follow @IonicFramework">
+  </a>
+  <a href="https://ionic.link/discord">
+    <img src="https://img.shields.io/discord/520266681499779082?color=7289DA&label=%23ionic&logo=discord&logoColor=white" alt="Official Ionic Discord" />
+  </a>
+</p>
+
+<h2 align="center">
+  <a href="https://ionicframework.com/getting-started/">Quickstart</a>
+  <span> · </span>
+  <a href="https://ionicframework.com/docs/">
+    Documentation
+  </a>
+  <span> · </span>
+  <a href="https://ionicframework.com/docs/contributing/how-to-contribute">Contribute</a>
```

Comment:
It looks like this page is missing some content from the contributing doc the original readme linked to: https://github.com/ionic-team/ionic-framework/blob/main/.github/CONTRIBUTING.md Maybe we should go with that for now?